Lib/enum.py

```diff
@@ -71,13 +86,45 @@ def _break_on_call_reduce(self, proto):
         setattr(obj, '__reduce_ex__', _break_on_call_reduce)
         setattr(obj, '__module__', '<unknown>')
 
+def _iter_bits_lsb(num):
+    while num:
+        b = num & (~num + 1)
+        yield b
+        num ^= b
+
+def bin(num, max_bits=None):
+    """
+    Like built-in bin(), except negative values are represented in
+    twos-compliment, and the leading bit always indicates sign
+    (0=positive, 1=negative).
+
+    >>> bin(10)
+    '0b0 1010'
+    >>> bin(~10)   # ~10 is -11
+    '0b1 0101'
```

Comment:
the space delimiter is quite strange-- I'm skeptical that it's going to help anybody

Reply:
It keeps the sign bit separate from the flag bits.  It helps me, at least.  At some point I hope to have the flag portion in nibbles.